README.md

```diff
@@ -41,17 +41,14 @@ Test Matrix
 ------------
 | Andrid Studio Version | cmake in SDK| NDK      | Result  |
 |-----------------------|-------------|----------|---------|
-| 2.2.0-beta1           | 3.6.3133135 | NDK-r12  | Yes     |
-| 2.2.0-beta1           | 3.6.3133135 | r13-beta1| NO      |
+| 3.0.0                 | 3.6.4111459 | NDK-r16  | Yes     |
 
 Known Issue:
- - r13-beta layer source needed to be updated
- - Studio Beta cmake 3.6.3133135 having issues with r13-beta
 
 Getting Started
 ---------------
 1. [Download Android Studio](http://tools.android.com/download/studio/canary)
```

Comment:
I would remove the first two steps, as you've already included them as prerequisites.